### PR TITLE
Update Libraries version

### PR DESCRIPTION
**Pull request by Sarbjyotsingh**

### CODE REVIEW

File: Libraries.txt [hunk 1]
```diff
@@ -0,0 +1,915 @@
+Open Source libraries used in the Scanbot Barcode Scanner SDK for Cordova version 3.0.1:
+
+------------------------------------------------------------------------------------------------------------------------
+
+Open Source Libraries used in the Scanbot Barcode Scanner SDK for Android Version 3.0.1:
```
Comment (rutkayb) [inline Libraries.txt:5]: Should be full sdk instead of barcode sdk

File: Libraries.txt [hunk 2]
```diff
@@ -0,0 +1,915 @@
+Open Source libraries used in the Scanbot Barcode Scanner SDK for Cordova version 3.0.1:
```
Comment (rutkayb) [inline Libraries.txt:1]: Cordova version is not right

**Review by rutkayb — CHANGES_REQUESTED** [review]

File: Libraries.txt (hunk 1 above)
Comment (Sarbjyotsingh) [reply Libraries.txt]: Fixed

File: Libraries.txt (hunk 2 above)
Comment (Sarbjyotsingh) [reply Libraries.txt]: Fixed

**Review by rutkayb — APPROVED** [review]

**Pull request merged**